src/fragments/lib/geo/android/search/10_search_by_text.mdx

```diff
@@ -0,0 +1,70 @@
+### Search by text
+
+The `Amplify.Geo.searchByText()` API enables you to search for places or points of interest by free-form text, such as an address, name, city, or region.
+
+<BlockSwitcher>
+<Block name="Java">
+
+```java
+String searchQuery = "Amazon Go";
+Amplify.Geo.searchByText(searchQuery,
+    result -> {
+        for (final Place place : result.getPlaces()) {
+            Log.i("MyAmplifyApp", place.toString());
+        }
+    },
+    error -> Log.e("MyAmplifyApp", "Failed to search for " + searchQuery, error)
+);
+```
+
+</Block>
+<Block name="Kotlin">
```

Comment:
Is Kotlin Coroutine and RxJava supported?

Reply:
not yet